src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -360,9 +360,9 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_cos)
 
 // tan
 MXNET_OPERATOR_REGISTER_UNARY(tan)
-.describe(R"code(Tangent, element-wise.
+.describe(R"code(Returns element-wise trigonometric tangent value of the input.
 
-Then input is in radians (:math:`2\pi` rad equals 360 degress).
+The input is in radians (:math:`2\pi` rad equals 360 degrees).
```

Comment:
Same: 

"The input should be specified in radians (:math:2\pi rad equals 360 degrees)."